great_expectations/rule_based_profiler/parameter_container.py

```diff
@@ -359,31 +359,23 @@ def _build_parameter_node_tree_for_one_parameter(
     parameter_value: Any,
 ) -> None:
     """
-    Recursively builds a tree of ParameterNode objects, creating new ParameterNode objects parsimoniously (i.e., only if
-    ParameterNode object, corresponding to a part of fully-qualified parameter names in a "name space" does not exist).
+    Builds a tree of ParameterNode objects.
     :param parameter_node: root-level ParameterNode for the sub-tree, characterized by the first parameter name in list
     :param parameter_name_as_list: list of parts of a fully-qualified parameter name of sub-tree (or sub "name space")
     :param parameter_value: value pertaining to the last part of the fully-qualified parameter name ("leaf node")
     """
-    parameter_name_part: str = parameter_name_as_list[0]
-
-    # If the fully-qualified parameter name (or "name space") is still compound (i.e., not at "leaf node" / last part),
-    # then build the sub-tree, creating the descendant ParameterNode (to hold the sub-tree), if no descendants exist.
-    if len(parameter_name_as_list) > 1:
-        if parameter_name_part not in parameter_node:
-            parameter_node[parameter_name_part] = ParameterNode({})
-
-        _build_parameter_node_tree_for_one_parameter(
-            parameter_node=parameter_node[parameter_name_part],
-            parameter_name_as_list=parameter_name_as_list[1:],
-            parameter_value=parameter_value,
-        )
-    else:
-        # If the fully-qualified parameter name (or "name space") is trivial (i.e., at "leaf node" / last part), then
-        # store the supplied attribute value into the given ParameterNode using leaf "parameter_name_part" name as key.
```

Comment:
Would you mind adding this comment back where appropriate? I believe it now applies at line 376? At least I'm sure most of the comment applies.

Reply:
Perfect! ✅